tests/test_resource_api.py

```diff
@@ -72,31 +90,34 @@ def test_access_resource(self):
         self.assertEqual(resp.status_code, 200)
 
         with self.subTest("Permission is granted"):
-            resp = self.app.get(f'/v1/resource/{rt_name}', headers=user_header)
+            resp = self.app.get(f'/v1/resource/{test_resource}', headers=user_header)
             self.assertEqual(resp.status_code, 200)
 
     def test_get_resource(self):
         """Pages of resource are retrieved when using the get resource API"""
         for i in range(11):
-            self.app.post(f'/v1/resource/tr{i}', data=json.dumps({'actions': ['tr:action1']}), headers=admin_headers)
+            self.app.post(f'/v1/resource/{resource_type_name()}', data=json.dumps({'actions': ['tr:action1']}),
```

Comment:
```suggestion
            self.app.post(f'/v1/resource/{test_resource}', data=json.dumps({'actions': ['tr:action1']}),
```

Reply:
This loop is supposed to generate multiple resource types. Doing this would only generate a single resource type.

Reply:
Ok - thanks for clarifying!